cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/MessagesApiModel.kt

```diff
@@ -156,6 +156,8 @@ data class USNatConsentStatus(
     @SerialName("consentedToAny") val consentedToAny: Boolean?,
     @SerialName("granularStatus") val granularStatus: USNatGranularStatus?,
     @SerialName("hasConsentData") val hasConsentData: Boolean?,
+    @SerialName("legalBasisChanges") var legalBasisChanges: Boolean? = null,
```

Comment:
@carmelo-iriti I think it's safe to remove legalBasisChangeDate for now since it won't be used on usnat.